workos/types/user_management/authentication_response.py

```diff
@@ -27,6 +28,7 @@ class AuthenticationResponse(_AuthenticationResponseBase):
     authentication_method: Optional[AuthenticationMethod] = None
     impersonator: Optional[Impersonator] = None
     organization_id: Optional[str] = None
+    oauth_credentials: Optional[OauthCredentials] = None
```

Comment:
Will OAuth credentials exist for all authentication methods or just [authenticate with code](https://workos.com/docs/reference/user-management/authentication/code)?